content/account/app/stats.textile

```diff
@@ -0,0 +1,51 @@
+---
+title: Stats
+meta_description: “Monitor and analyze your app's performance with Ably's dashboard. Access realtime stats and trends for optimized management."
+meta_keywords: "Ably dashboard, dashboard, app performance, realtime stats, usage monitoring, performance analysis, data trends, statistics chart, statistics table"
+---
+
+The stats tab is an interface to monitor your app's performance and usage via the "statistics table":#table and "statistics chart":#chart.
+
+h2(#table). Statistics table
+
+The statistics table shows a summary of your app's activity and performance metrics:
+
+<a href="@content/screenshots/dash/stats-table.png" target="_blank">
+  <img src="@content/screenshots/dash/stats-table.png" style="width: 100%" alt="Your stats table">
+</a>
+
+The following explains how to use the statistics table:
+
+*Monitor apps in realtime* _The table supports realtime monitoring by displaying statistics for the last hour and minute, enabling you to identify and respond to issues quickly._
+
+*Track usage* _Monitor the number of messages published and received, presence events, and data transferred. This helps you understand your app's usage patterns._
+
+*Analyze performance* _View key performance indicators, such as peak connections and channels. These insights help you gauge your app's capacity and performance over time._
+
+*Analyze trends* _Compare data across different periods (e.g. last month vs. this month) to identify trends, spikes, and periods of low usage. Use this information to make informed decisions about resource allocation and scaling._
+
+*Track events* _Track specific events, such as webhook executions, queue operations, and push notifications, to understand better and debug your app's functionalities._
+
+*Plan* _Use insights into peak usage times and data transfer volumes to plan for capacity, ensuring your app can handle expected loads without performance issues._
+
+*Forecast* _Review forecast data to anticipate future usage, helping you optimize server resources and improve the user experience._
+
+*Download insights* _Download or view detailed app statistics for a more granular analysis, enabling deeper dives into specific metrics as needed._
+
+*Generate summary reports* _Generate concise summary reports of key metrics to communicate app performance and usage statistics to stakeholders._
```

Comment:
These make it seem like they're UI interactions, whereas other than 'download', aren't these all just synonyms for viewing the table data?